web/src/infra/common/components/base/FreeBoardGamesBar.tsx

```diff
@@ -1,55 +1,102 @@
+/* eslint-disable react/prop-types */
```

Comment:
Since FBG uses TypeScript, I feel like `react/prop-types` might be a rule we want to disable project-wide.

Reply:
I agree, done.